#! /bin/sh /usr/share/dpatch/dpatch-run
## 23_sccp_crash_fix.dpatch by  <eblircz@oceanus.sz13.dyndns.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix uninitialized variable to prevent crash

@DPATCH@

Index: trunk/epan/dissectors/packet-sccp.c
===================================================================
--- trunk/epan/dissectors/packet-sccp.c	(revision 28057)
+++ trunk/epan/dissectors/packet-sccp.c	(revision 28058)
@@ -805,11 +805,12 @@
 	a->called_ssn = INVALID_SSN;
 	a->has_fw_key = FALSE;
 	a->has_bw_key = FALSE;
+	a->msgs = NULL;
+	a->curr_msg = NULL;
 	a->payload = SCCP_PLOAD_NONE;
 	a->calling_party = NULL;
 	a->called_party = NULL;
 	a->extra_info = NULL;
-	a->msgs = NULL;
 
 	return a;
 }
